### PR TITLE
Tc 32 henry bustamante 02

### PR DESCRIPTION
**Pull request by henrybc08**
another pull request to move on a separate call (step) save button for accident form

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        CommonEvents.setInputField(accidentDaysTransferredField, "2");
+        CommonEvents.selectOptionFieldByValue(saCategoryListField, "2");
+        CommonEvents.selectOptionFieldByValue(saTypeListField, "2");
+
+    }
+
+    public void clickSaveAccidentForm() {
+        CommonEvents.clickButton(saveButton);
+    }
+
```
Comment (lizeth-salazar) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java:91]: Please remove this space

File: src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+/*
+Created on 25/6/18
+@author: HENRYBC
+*/
+public class AccidentForm extends BasePage {
+
```
Comment (lizeth-salazar) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java:14]: Please add constructor for this class

File: src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    public void fillAccidentForm(){
+        CommonEvents.selectOptionFieldByValue(employeeList, "6");
+        CommonEvents.setInputField(accidentDescriptionField, "new accident");
+        CommonEvents.setDateField(accidentDateField, "06152018");
+        CommonEvents.setInputField(accidentWhereField, "it happened at home");
+        CommonEvents.setInputField(accidentDaysOutWorkField, "3");
+        CommonEvents.setInputField(accidentDaysTransferredField, "2");
+        CommonEvents.selectOptionFieldByValue(saCategoryListField, "2");
+        CommonEvents.selectOptionFieldByValue(saTypeListField, "2");
+
```
Comment (lizeth-salazar) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java]: Remove this blank space

File: src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    private WebElement saCategoryListField;
+
+    @FindBy(id = "saTypeListFieldId")
+    private WebElement saTypeListField;
+
+    @FindBy(id = "saveButtonId")
+    private WebElement saveButton;
+
+    public void fillAccidentForm(){
+        CommonEvents.selectOptionFieldByValue(employeeList, "6");
```
Comment (lizeth-salazar) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/accident/AccidentForm.java]: Every single line should be a specific method to add values into a field

File: src/test/resources/features/accident/CreateAccident.feature [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+Feature: CreateAccident
+  Scenario: new accident created is displayed on list of accidents when we clicked on 'save' button
+    Given  'SSI-A' home page is loaded
+    And click Safety 'menu' on 'Header' page
+    And click 'Accident' sub menu on 'Safety' menu
+    And click 'New accident for Employee' button on 'Accident' list page
+    And fill all information required for 'Accident' form
+    And click 'save' button on 'Accident' form
+    Then 'Accident list' page loads with new records added
+    #Then 'Accident list' page loads correctly
```
Comment (lizeth-salazar) [inline src/test/resources/features/accident/CreateAccident.feature]: Remove this commented line

File: src/test/resources/features/accident/CreateAccident.feature [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# @Author: Henry Bustamante
+# Feature: createAccident
+
+Feature: CreateAccident
+  Scenario: new accident created is displayed on list of accidents when we clicked on 'save' button
+    Given  'SSI-A' home page is loaded
+    And click Safety 'menu' on 'Header' page
```
Comment (lizeth-salazar) [inline src/test/resources/features/accident/CreateAccident.feature]: 'Safety' should be into the single quotes

**Review by lizeth-salazar — CHANGES_REQUESTED** [review]
Please review the comments

**Comment by henrybc08** [comment]
captura de pantalla 2018-06-30 a las 16 30 27
captura de pantalla 2018-06-30 a las 16 25 37
All observations were corrected

**Pull request merged**